cvmfs/sync_item.h

```diff
@@ -39,34 +39,38 @@ enum SyncItemType {
  * details.
  */
 class SyncItem {
```

Comment:
The usual order is a bit mixed up: we normally have friend classes first, then public, protected, private